quartz/components/pages/FolderContent.tsx

```diff
@@ -49,11 +49,13 @@ export default ((opts?: Partial<FolderContentOptions>) => {
         <article>
           <p>{content}</p>
         </article>
-        {options.showFolderCount && (
-          <p>{pluralize(allPagesInFolder.length, "item")} under this folder.</p>
-        )}
-        <div>
-          <PageList {...listProps} />
+        <div class="folder-content-hint">
```

Comment:
```suggestion
        <div class="page-listing">
```

with this change, can you also do the same treatment for both usages of `<PageList>` in `TagContent.tsx`